yarn.lock

```diff
@@ -195,7 +195,7 @@
     semver "^5.4.1"
```

Comment:
Please make sure to rebase and regenerate this file to minimize changes because only 1 single dependency touched in this PR, so it should not introduce loads of changes in the lock file.